server/service/individualServices/LinkAnalysis/PrepareSwitchRedundantTransmitterPairOffForwarding.js

```diff
@@ -0,0 +1,371 @@
+        requestId: switchOffTXPairRequestId,
+        statusOfLink: statusOfLink
+    });
+    EventDispatcherWithResponse.BuildAndTriggerRestRequestToRequestor(requestorProtocol, requestorAddress, requestorPort, requestorReceiveOperation, requestHeaders, body, "POST", traceIndicatorIncrementer++);
+}
+
+/**
+ *  Checks for the request id in the  switchOffTXPairTransactionMap if exists proceeds with the execution of callback
+ *  RequestForSwitchingRedundantTransmitterPairOffInitiatesTransaction.Response otherwis just returns.
+ * @param {String}  provideTransmitterStatusOfParallelLinksRequestId request ID that is generated during provideTransmitterStatusOfParallelLinks.
+ * @param {Object}  parallelLinkList contains parallel links list .
+ * @param {Object}  requestHeaders Holds information of the requestHeaders like Xcorrelator , CustomerJourney,User etc.
+ */
+exports.receiveTransmitterStatusOfParallelLinks = async function (provideTransmitterStatusOfParallelLinksRequestId, parallelLinkList, requestHeaders) {
+    let switchOffTXPairRequestorDetails = {};
+    let switchOffTXPairRequestId = switchOffTXPairTransactionMap.get(provideTransmitterStatusOfParallelLinksRequestId);
+    if (switchOffTXPairRequestId && switchOffTXPairRequestId != "") {
+        switchOffTXPairRequestorDetails = switchOffTXPairRequestMap.get(switchOffTXPairRequestId);
+    }
+    if (Object.keys(switchOffTXPairRequestorDetails).length > 0) {
+        requestHeaders.traceIndicator = switchOffTXPairRequestorDetails.traceIndicator;
+        traceIndicatorIncrementer = switchOffTXPairRequestorDetails.traceIndicatorIncrementer;
+        responseOfSwitchOffTXPairRequestor(switchOffTXPairRequestId, switchOffTXPairRequestorDetails, parallelLinkList, requestHeaders, traceIndicatorIncrementer)
+    }
+
+    /******************************************************************************************************************
+     *  Delete data from switchOffTXPairRequestMap and switchOffTXPairTransactionMap
+     *****************************************************************************************************************/
+
+    switchOffTXPairRequestMap.delete(switchOffTXPairRequestId);
+    switchOffTXPairTransactionMap.delete(provideTransmitterStatusOfParallelLinksRequestId);
+    return;
+}
+
+/**
+ * Formulates status of link
+ * @param {List}    parallelLinks contains parallel links list .
+ * @param {String}  linkId Id of the initial link. 
+ * @param {Object}  requestHeaders Holds information of the requestHeaders like Xcorrelator , CustomerJourney,User etc.
+ * @param {Integer} traceIndicatorIncrementer traceIndicatorIncrementer to increment the trace indicator
+ */
+async function formulateStatusOfLink(parallelLinks, intialLinkId, requestHeaders, traceIndicatorIncrementer) {
+    let statusOfLink = "UndefinedState";
+    let statusOfLinkAndTraceIndicatorIncrementer = {};
+    if (parallelLinks.length === 2) {
```

Comment:
As per diagram, [switchRedundantTransmitterIsOff](https://raw.githubusercontent.com/openBackhaul/AirInterfacePowerSaver/develop/spec/diagrams/05_LinkRelatedSwitchingOperation/500_switchRedundantTransmitterPairOff.png), the check shall be included for parallelLinks>=2
![image](https://github.com/openBackhaul/AirInterfacePowerSaver/assets/88484011/afb5cb4e-5575-4446-94be-c5be7827a889)

Reply:
Also, we need to check once if it is >=2 or ==2

Reply:
Changed the check as per diagram . But , as per discussion with testing team there will not be links >2

Reply:
@PrathibaJee , @vanithavalluripalli9  shall we create an issue for updating this concept in diagram